test/test_utils.sh

```diff
@@ -174,14 +174,16 @@ helm_install_fission() {
     echo "Deleting old releases"
     helm list -q|xargs -I@ bash -c "helm_uninstall_fission @"
 
+    # deleting ns does take a while after command is issued.
+    sleep 45
```

Comment:
this is a pretty long wait to be doing unconditionally, something like this might be better?
```
while kubectl get ns | grep $ns # or $fns i'm not sure
do 
    sleep 5
done
```